pkg/scorecard.go

```diff
@@ -105,9 +105,25 @@ func RunScorecard(ctx context.Context,
 	defer repoClient.Close()
 
 	commitSHA, err := getRepoCommitHash(repoClient)
-	if err != nil || commitSHA == "" {
+
+	if errors.Is(err, sce.ErrorEmptyRepository) {
+		versionInfo := version.GetVersionInfo()
+		ret := ScorecardResult{
+			Repo: RepoInfo{
+				Name:      repo.URI(),
+				CommitSHA: commitSHA,
+			},
+			Scorecard: ScorecardInfo{
+				Version:   versionInfo.GitVersion,
+				CommitSHA: versionInfo.GitCommit,
+			},
+			Date: time.Now(),
+		}
+		return ret, nil
```

Comment:
If we're not surfacing the error, let's not add it to the public error list. A private error at the top of this file should work too?

I was curious, so if we return the error we get something like this instead:
```
Error: RunScorecard: repository empty
2023/08/25 10:42:40 error during command execution: RunScorecard: repository empty
exit status 1
```

Reply:
changed this to a local error

Reply:
can you make it private? e.g `errEmptyRepository`